ui-v2/app/router.js

```diff
@@ -1,58 +1,98 @@
-    });
+      show: {
+        _options: { path: '/:name' },
+      },
+    },
     // Nodes represent a consul node
-    this.route('nodes', { path: '/nodes' }, function() {
+    nodes: {
+      _options: { path: '/nodes' },
       // Show an individual node
-      this.route('show', { path: '/:name' });
-    });
+      show: {
+        _options: { path: '/:name' },
+      },
+    },
     // Intentions represent a consul intention
-    this.route('intentions', { path: '/intentions' }, function() {
-      this.route('edit', { path: '/:id' });
-      this.route('create', { path: '/create' });
-    });
+    intentions: {
+      _options: { path: '/intentions' },
+      edit: {
+        _options: { path: '/:id' },
+      },
+      create: {
+        _options: { path: '/create' },
+      },
+    },
     // Key/Value
-    this.route('kv', { path: '/kv' }, function() {
-      this.route('folder', { path: '/*key' });
-      this.route('edit', { path: '/*key/edit' });
-      this.route('create', { path: '/*key/create' });
-      this.route('root-create', { path: '/create' });
-    });
+    kv: {
+      _options: { path: '/kv' },
+      folder: {
+        _options: { path: '/*key' },
+      },
+      edit: {
+        _options: { path: '/*key/edit' },
+      },
+      create: {
+        _options: { path: '/*key/create' },
+      },
+      'root-create': {
+        _options: { path: '/create' },
+      },
+    },
     // ACLs
-    this.route('acls', { path: '/acls' }, function() {
-      this.route('edit', { path: '/:id' });
-      this.route('create', { path: '/create' });
-      this.route('policies', { path: '/policies' }, function() {
-        this.route('edit', { path: '/:id' });
-        this.route('create', { path: '/create' });
-      });
-      this.route('tokens', { path: '/tokens' }, function() {
-        this.route('edit', { path: '/:id' });
-        this.route('create', { path: '/create' });
-      });
-    });
-  });
-
+    acls: {
+      _options: { path: '/acls' },
+      edit: {
+        _options: { path: '/:id' },
+      },
+      create: {
+        _options: { path: '/create' },
+      },
+      policies: {
+        _options: { path: '/policies' },
+        edit: {
+          _options: { path: '/:id' },
+        },
+        create: {
+          _options: { path: '/create' },
+        },
+      },
+      tokens: {
+        _options: { path: '/tokens' },
+        edit: {
+          _options: { path: '/:id' },
+        },
+        create: {
+          _options: { path: '/create' },
+        },
+      },
+    },
+  },
   // Shows a datacenter picker. If you only have one
   // it just redirects you through.
-  this.route('index', { path: '/' });
-
+  index: {
+    _options: { path: '/' },
+  },
   // The settings page is global.
-  // this.route('settings', { path: '/settings' });
-  this.route('notfound', { path: '/*path' });
-});
-
-export default Router;
+  // settings: {
+  //   _options: { path: '/setting' },
+  // },
```

Comment:
@kaxcode here's that thing you spotted yesterday! Notice I must have mistakenly changed it from `settings` to `setting` at some point 😬 !!  I'm still trying to remember/find why its commented out here and not commented out currently.

I want to bounce around ideas for changing this anyway, ideally the URL would be `/~nspace/dc-1/somewhere/in/the/ui#settings` as the settings page is almost a modal, I remember trying this at the very start and remember that ember didn't like using `#`s in URLs. I have bigger ember muscles now though so maybe we could try this. 

Another option would be to treat the settings page in the same way that we treat all the other URLs so just `/~nspace/dc-1/settings`. That way we keep all the info we need in the URL and not in localStorage.

Lets chat later